### ISSUE TITLE
Accessibility regression: The "Add new tag" field has no focus style

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#23990]
**Describe the bug**

In WordPress 5.5 Beta 2, the "Add new tag" field in the Sidebar has no focus style. At least, has no focus style that can be perceived.

Screenshot:

Normal state:

<img width="345" alt="Screenshot 2020-07-16 at 15 23 49" src="https://user-images.githubusercontent.com/1682452/87676092-7750ac00-c778-11ea-9260-aa43ea3ea812.png">

Focus state:

<img width="353" alt="Screenshot 2020-07-16 at 14 58 02" src="https://user-images.githubusercontent.com/1682452/87676107-7d468d00-c778-11ea-86fc-3f0b28d553a6.png">

In WordPress 5.4, the focus style is clearly visible:

<img width="351" alt="Screenshot 2020-07-16 at 15 20 42" src="https://user-images.githubusercontent.com/1682452/87676131-846d9b00-c778-11ea-8073-a70ff8c0e388.png">
